### DISCUSSION TITLE
Add a "Favorite Skins" feature

### DISCUSSION THREAD

**Discussion by jhk2601** [ppy/osu#30355] [Ideas] [answered]
Many people have sprawling skin lists containing 50+ skins, most of which they never use. This is why the changing of the naming structure caused so much friction, since people are used to scrolling down an exact amount to find the 3 or 4 skins they actually use. The idea here would be an option to "favorite" a skin, causing it to always appear at the top of the dropdown, as well as a key bind to switch between only favorited skins (similar to alt-shift-r for a random skin). I was going to try to do this myself but realm and UI considerations mean I would probably do more harm than good. I think this would be a widely beloved QOL feature if added, and maybe even something you could supporter-lock.

**Comment by Joehuu** [comment, marked as answer]
May be implemented in https://github.com/ppy/osu/pull/36260, no keybind though.
